x-pack/plugins/transform/public/app/hooks/use_api.ts

```diff
@@ -85,6 +97,24 @@ export const useApi = () => {
       getIndices(): Promise<EsIndex[]> {
         return http.get(`/api/index_management/indices`);
       },
+      getHistogramsForFields(
+        indexPatternTitle: string,
+        fields: FieldHistogramRequestConfig[],
+        query: string | SavedSearchQuery,
+        samplerShardSize = DEFAULT_SAMPLER_SHARD_SIZE
+      ) {
+        const body = JSON.stringify({
+          query,
+          fields,
+          samplerShardSize,
+        });
+
+        return http.post({
+          path: `${API_BASE_PATH}field_histograms/${indexPatternTitle}`,
+          method: 'POST',
```

Comment:
do you need to set a method when you use `http.post`?

Reply:
Good catch! Missed it when copying over the code from ML to transforms, the calls are structured in a different way. Fixed in 5b2ecdc.